scripts/BONSpecialGenerator.swift

```diff
@@ -66,6 +67,7 @@ let charactersRequiringFormatStrings: Set<unichar> = [
 // These special characters are to be excluded from the human readable string dictionary.
 let charactersToExcludeFromHumanReadableStringDictionary: Set<unichar> = [
     0x0020,
+    0x202D
```

Comment:
#? Do we really want to exclude this from human readable strings? If this character is present in a string I'm debugging, I feel like I'd want to know about it.